opentok-angular.js

```diff
@@ -311,3 +293,94 @@ ng.module('opentok', [])
       };
     }
   ]);
+
+  const captionSubscriberTracker = (OTSession, $rootScope) => {
+    const MAX_SUBS_ON_SCREEN = 4;
+    const CAPTIONS_TIMEOUT = 5 * 1000;
```

Comment:
I'm guessing this is in units of milliseconds?  If so, may `CAPTIONS_TIMEOUT` -> `CAPTIONS_TIMEOUT_MSEC`? 🤷‍♂️

Reply:
Sounds good to me. The 5 second timer is arbitrary as well if you feel like a different amount of time would be better please let me know.